husky_description/package.xml

```diff
@@ -19,14 +19,13 @@
   <license>BSD</license>
 
   <buildtool_depend>catkin</buildtool_depend>
-  <build_depend>roslaunch</build_depend>
   <run_depend>robot_state_publisher</run_depend>
   <run_depend>urdf</run_depend>
   <run_depend>xacro</run_depend>
   <run_depend>ur_description</run_depend>
   <run_depend>lms1xx</run_depend>
 
-
+  <test_depend>roslaunch</test_depend>
```

Comment:
@mikepurvis I think `test_depend` is already defined also in format 1. I think this should be better.